venia-integration-tests/src/actions/accountAccess/index.js

```diff
@@ -0,0 +1,77 @@
+import {
+    accountTriggerButton,
+    createAccountInitiateButton,
+    firstNameTextField,
+    lastNameTextField,
+    createAccountEmailTextField,
+    createAccountPasswordTextField,
+    createAccountSubmitButton
+} from '../../fields/accountAccess';
+
+/**
+ * Utility function to open the login dialog
+ */
+export const openLoginDialog = () => {
+    // open the signin dialog
+    cy.get(accountTriggerButton).click();
+};
+
+/**
+ * Utility function to create account in Venia using the
+ * username and password provided.
+ *
+ * @param {String} firstName firstName to createAccount
+ * @param {String} lastName lastName to createAccount
+ * @param {String} accountEmail accountEmail to createAccount
+ * @param {String} accountPassword accountPassword to createAccount
+ */
+export const createAccount = (
```

Comment:
I dont think we need these any more. We moved these to custom cypress actions. Or am I missing something?